frontend/app/routes/$lang+/_protected+/personal-information+/index.tsx

```diff
@@ -123,7 +123,10 @@ export default function PersonalInformationIndex() {
           )}
         </DescriptionListItem>
         <DescriptionListItem term={t('personal-information:index.phone-number')}>
-          <p>{personalInformation.primaryTelephoneNumber}</p>
+          <dt className="font-semibold">{t('personal-information:index.primary-phone-number')}</dt>
+          <p>{personalInformation.primaryTelephoneNumber ?? 'N/A'}</p>
+          <dt className="font-semibold">{t('personal-information:index.alternate-phone-number')}</dt>
+          <p>{personalInformation.alternateTelephoneNumber ?? 'N/A'}</p>
```

Comment:
These newly added lines will fail HTML validation because a `<dt>` must be in a `<dl>`. We should also be using the corresponding `<dd>` instead of `<p>`:

```html
 <dl>
  <dt className="font-semibold">{t('personal-information:index.primary-phone-number')}</dt>
  <dd>{personalInformation.primaryTelephoneNumber ?? 'N/A'}</dd>
  <dt className="font-semibold">{t('personal-information:index.alternate-phone-number')}</dt>
  <dd>{personalInformation.alternateTelephoneNumber ?? 'N/A'}</dd>
</dl>
```

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

Reply:
done